source/common/upstream/eds.cc

```diff
@@ -42,7 +43,10 @@ EdsClusterImpl::EdsClusterImpl(const envoy::api::v2::Cluster& cluster, Runtime::
       "envoy.api.v2.EndpointDiscoveryService.StreamEndpoints");
 }
 
-void EdsClusterImpl::startPreInit() { subscription_->start({cluster_name_}, *this); }
+void EdsClusterImpl::startPreInit() { 
+	ENVOY_LOG(info, "Doug: EdsClusterImpl::startPreInit() {}", cluster_name_);
```

Comment:
Is there any reason that these "debug" loggings are not printed as "ENVOY_LOG(debug, ... " ?

Reply:
Yes, the debug level logging in Envoy is very verbose, so I wanted to see these log entries on a less noisy level. However, I will be removing all of the log statements I added in this PR. They were just to help me understand the control flow while debugging this project.